integration/spark/src/test/java/io/openlineage/spark/agent/SparkContainerIntegrationTest.java

```diff
@@ -94,6 +94,24 @@ private static MockServerContainer makeMockServerContainer() {
         .withCommand(command);
   }
 
+  private static GenericContainer<?> makePysparkContainerWithDefaultConf(
+      String namespace, String command) {
+    return makePysparkContainer(
+        "--master",
+        "local",
+        "--conf",
+        "spark.openlineage.host=" + "http://openlineageclient:1080",
```

Comment:
_Minor_: Mind if we use a more reasonable http port like `8080`? (used as the default API port for Marquez)

Reply:
This is connecting to MockServer, and I unfortunately don't see easy way to change that via env variables. 

https://www.mock-server.com/mock_server/configuration_properties.html

Reply:
Ahh, actually, that makes total sense. You can disregard this comment ;)